font/face.go

```diff
@@ -1,13 +1,16 @@
 package font
 
-import "github.com/benoitkugler/textlayout/fonts"
-
-// make sure that we can use textlayout/fonts.Face as Face
-var _ Face = (fonts.Face)(nil)
+import (
+	"github.com/benoitkugler/textlayout/fonts"
+	"github.com/benoitkugler/textlayout/harfbuzz"
+)
 
 type GID = fonts.GID
+type GlyphMask = harfbuzz.GlyphMask
 
 type Face interface {
+	fonts.Face // make sure that we can use textlayout/fonts.Face as Face
```

Comment:
Minor comment : I was thinking to `textlayout/fonts.Face` as a draft interface, that we will polish when building go-text. 
(When the API is settled, it is not obvious that the methods in  `textlayout/fonts.Face` are all needed.)

Anyway, if we prefer to stick to `textlayout/fonts.Face`, we could simply use 
`type Face = textlayout/fonts.Face`

Reply:
I started with a `Face` alias - but then `NominalGlyph` was added, and I didn't think we wanted two different face interfaces in this package?

Reply:
Alright. Since the `NominalGlyph` is already in `textlayout/fonts.Face`, I then suggest to use `type Face = textlayout/fonts.Face`